src/EditorFeatures/Test/InheritanceMargin/InheritanceMarginTests.cs

```diff
@@ -989,6 +989,65 @@ public class {{|target1:Bar2|}} : IBar{lessThanToken}int{greaterThanToken}, IBar
                 itemForFooInBar2);
         }
 
+        [Fact]
+        public Task TestCSharpExplicitInterfaceImplementation()
+        {
+            var lessThanToken = SecurityElement.Escape("<");
+            var greaterThanToken = SecurityElement.Escape(">");
```

Comment:
what the heck is this?

Reply:
That's the escaped character for '<' and '>' for xml.

Reply:
Oops, l mean  `&lt;` and `&gt;`. Github also rendering these characters.